hack/terraform/aws/delete_dependency.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Delete security groups and the dependencies on AWS created by users.
+vpc_id="$( terraform output vpc_id | sed 's/\"//g' )"
+region="$( terraform output region | sed 's/\"//g' )"
+
+security_groups="$( aws ec2 describe-security-groups --region $region --filters Name=vpc-id,Values=$vpc_id )"
```

Comment:
can't you fetch terraform created secruity group using --filter option specifying tag field